src/main/java/com/adobe/target/edge/client/service/TelemetryService.java

```diff
@@ -66,11 +69,20 @@ public TelemetryEntry createTelemetryEntry(
     com.adobe.target.delivery.v1.model.DecisioningMethod decisioningMethod =
         com.adobe.target.delivery.v1.model.DecisioningMethod.valueOf(
             getDecisioningMethod(targetDeliveryRequest).name());
-    TelemetryFeatures telemetryFeatures =
-        new TelemetryFeatures().decisioningMethod(decisioningMethod);
+
+    TelemetryFeatures telemetryFeatures = new TelemetryFeatures();
+    telemetryFeatures.setDecisioningMethod(decisioningMethod);
+    telemetryFeatures.setExecuteMboxCount(executeMboxCount(targetDeliveryRequest));
```

Comment:
For these counts and boolean values let's only call the setter if the value is truthy.  If the value is omitted it can be assumed to be falsy and the backend will fill in the value before persisting it to Splunk.   One approach that would cut down on the amount of code would be to have a `buildTelemetryFeatures` method that creates a TelemetryFeatures instance, calls whichever setters are necessary, and returns it.